### DISCUSSION TITLE
Classification repository?

### DISCUSSION THREAD

**Discussion by peterkobylarz** [earthlab-education/earth-analytics-02-python-2024#8] [Q&A02 Land Cover Classification]
@eculler I may be missing something here but I can't find the link to accept the Classification assignment on GitHub Classroom. All I can see on Canvas is the "Portfolio: Classification" assignment. Can someone share the link with me? Thanks!

**Comment by edchanonline** [comment]
It hasn't been posted yet. Elsa mentioned she's behind on posting this assignment
